Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -23,6 +23,14 @@ enum BlockList: String {
     }
 }
 
+@available(iOS 11.0, *)
+enum BlockerStatus: String {
+    case Disabled
+    case NotBlocking // When TP is enabled but nothing is being blocked
+    case Whitelisted
```

Comment:
If we have a `Whitelisted` icon state, do we also need a `TurnedOffForTab` for the per-tab setting?

Reply:
I made a whiteListed state but there isnt actually a whitelisted icon. The icon we would show for whitelisted and "turnedOffFortab" should be the same icon. (the shied with a strike through.)